blazor/treegrid/clipboard.md

```diff
@@ -384,4 +384,16 @@ N> To perform paste functionality, it requires the selection `Mode` to be `Cell`
 
 ### Limitations of Paste Functionality
 
-* Since the string values are not parsed to number and date type, so when the copied string type cells are pasted to number type cells then it will be displayed as **NaN**. For date type cells, when the copied string format cells are pasted to date type cells then it will be displayed as an **empty cell**.
\ No newline at end of file
+* Since the string values are not parsed to number and date type, so when the copied string type cells are pasted to number type cells then it will be displayed as **NaN**. For date type cells, when the copied string format cells are pasted to date type cells then it will be displayed as an **empty cell**.
+
+## Clipboard events
+
+Clipboard events are events that are triggered when performing a copy or paste action on the tree grid. 
+
+### Available clipboard events
+
+* [BeforeCopyPaste](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeGrid.TreeGridEvents-1.html#Syncfusion_Blazor_TreeGrid_TreeGridEvents_1_BeforeCopyPaste) : Triggered when the cell is copied or pasted.
```

Comment:
Triggered before the cell is copied or pasted in the tree grid cell and you can cancel this entire copy or paste action by using this event.

Reply:
Resolved by changing the sentence.